### PR TITLE
Add SECURITY document

### LINKED ISSUE #2: Add SECURITY.md

### PR DESCRIPTION
**Pull request by scottrigby**
Fixes #2 

@fluxcd/oversight-committee 

Initial draft of Flux security processes and info.

- Much of this drawn from https://kubernetes.io/docs/reference/issues-security/security/ and https://github.com/helm/community/blob/master/SECURITY.md. Simplified a bit. We can continue to provide further detail as needed.
- ~Temporary PLACEHOLDER for @hiddeco PGP keys.~
- We have a keybase team, which is a logical choice for trusting user account PGP keys
- Also are there other maintainers who would like to be initial security team members?

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Flux Security
+
+This document <https://github.com/fluxcd/community/blob/main/SECURITY.md> defines security reporting, handling, and disclosure information for the Flux CNCF project and community.
+
+## Report a Vulnerability
+
+We're very thankful for – and if desired happy to credit – security researchers and users who report vulnerabilities to the Flux Community.
```
Comment (hiddeco) [inline SECURITY.md:7]:
```suggestion
We're very thankful for – and if desired happy to credit – security researchers and users who report vulnerabilities to the Flux community.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Flux Security
+
+This document <https://github.com/fluxcd/community/blob/main/SECURITY.md> defines security reporting, handling, and disclosure information for the Flux CNCF project and community.
```
Comment (hiddeco) [inline SECURITY.md:3]:
```suggestion
This document <https://github.com/fluxcd/community/blob/main/SECURITY.md> defines security reporting, handling, and disclosure information for the Flux project and community.
```

(We removed "CNCF" references like this elsewhere).

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Crrent Security Team members:
+
+| Name | GitHub | Key URL | Fingerprint |
+| -- | -- | -- | -- |
+| Scott Rigby | [@scottrigby](https://github.com/scottrigby) | <https://keybase.io/r6by/pgp_keys.asc> | 208D D36E D5BB 3745 A167  43A4 C7C6 FBB5 B91C 1155 |
+| Hidde Beydals | [@hiddeco](https://github.com/hiddeco) | <https://keybase.io/hidde/pgp_keys.asc> | C910 7A9B 55A4 DD77 062B 9731 B6E3 6A6A C54A CD59 |
+
+## Handling
+
+- All reports are thoroughly investigated by the Flux security team.
```
Comment (hiddeco) [inline SECURITY.md:26]:
```suggestion
- All reports are thoroughly investigated by the Security Team.
```

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+- To make a report please email the private security list at <cncf-flux-security@lists.cncf.io> with the details.
+  We ask that reporters act in good faith by not disclosing the issue to others.
+- You may, but are not required to, encrypt your email to this list using the PGP keys of Security Team members, listed below.
+- The Security Team will fix the issue as soon as possible and coordinate a release date with you.
+- You will be able to choose if you want public acknowledgement of your effort and how you would like to be credited.
+
+## Security Team
+
+Crrent Security Team members:
```
Comment (squaremo) [inline SECURITY.md:17]:
```suggestion
Current Security Team members:
```

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+- As the security issue moves through the identification and resolution process, the reporter will be notified.
+- Additional questions about the vulnerability may also be asked of the reporter.
+
+## Disclosures
+
+Vulnerability disclosures are emailed to the Flux Dev mailing list <https://lists.cncf.io/g/cncf-flux-dev> and announced publicly.
+Disclosures will contain an overview, details about the vulnerability, a fix that will typically be an update, and optionally a workaround if one is available.
+
+We prefer to fully disclose the vulnerability as soon as possible once a user mitigation is available.
+Disclosures will be published on the same day as a release fixing the vulnerability after the release is published.
```
Comment (squaremo) [inline SECURITY.md:38]:
```suggestion
Disclosures will be published on the same day as a release fixing the vulnerability, after the release is published.
```

**Review by squaremo — APPROVED** [review]
Solid and sensible :+1:

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- The Security Team will fix the issue as soon as possible and coordinate a release date with you.
+- You will be able to choose if you want public acknowledgement of your effort and how you would like to be credited.
+
+## Security Team
+
+Crrent Security Team members:
+
+| Name | GitHub | Key URL | Fingerprint |
+| -- | -- | -- | -- |
+| Scott Rigby | [@scottrigby](https://github.com/scottrigby) | <https://keybase.io/r6by/pgp_keys.asc> | 208D D36E D5BB 3745 A167  43A4 C7C6 FBB5 B91C 1155 |
```
Comment (hiddeco) [inline SECURITY.md:21]:
```suggestion
| Scott Rigby | [@scottrigby](https://github.com/scottrigby) | <https://keybase.io/r6by/pgp_keys.asc> | 208D D36E D5BB 3745 A167 43A4 C7C6 FBB5 B91C 1155 |
```

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @scottrigby 🍰

**Review by hiddeco — APPROVED** [review]
Thank you @scottrigby :tada:

**Pull request merged**